From 74ab06bc284746df6def9aa87ffac6b4ff04db94 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Tue, 7 Sep 2010 13:35:16 +0000
Subject: [PATCH v2 08/20] gettextize: git-revert literal "me" messages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Redo the translation of the remaining messages that used the `me'
variable. These are all error messages referencing the command name,
so the name shouldn't be translated.

Reported-by: Jonathan Nieder <jrnieder@gmail.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/revert.c |   10 +++++++---
 1 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/builtin/revert.c b/builtin/revert.c
index ad2c76a..a89dd12 100644
--- a/builtin/revert.c
+++ b/builtin/revert.c
@@ -347,7 +347,8 @@ static int do_recursive_merge(struct commit *base, struct commit *next,
 	if (active_cache_changed &&
 	    (write_cache(index_fd, active_cache, active_nr) ||
 	     commit_locked_index(&index_lock)))
-		die(_("%s: Unable to write new index file"), _(me));
+		/* TRANSLATORS: %s will be "revert" or "cherry-pick" */
+		die(_("%s: Unable to write new index file"), me);
 	rollback_lock_file(&index_lock);
 
 	if (!clean) {
@@ -454,8 +455,10 @@ static int do_pick_commit(void)
 		return fast_forward_to(commit->object.sha1, head);
 
 	if (parent && parse_commit(parent) < 0)
+		/* TRANSLATORS: The first %s will be "revert" or
+		   "cherry-pick", the second %s a SHA1 */
 		die(_("%s: cannot parse parent commit %s"),
-		    _(me), sha1_to_hex(parent->object.sha1));
+		    me, sha1_to_hex(parent->object.sha1));
 
 	if (get_message(commit->buffer, &msg) != 0)
 		die(_("Cannot get commit message for %s"),
@@ -577,7 +580,8 @@ static int revert_or_cherry_pick(int argc, const char **argv)
 	}
 
 	if (read_cache() < 0)
-		die(_("git %s: failed to read the index"), _(me));
+		/* TRANSLATORS: %s will be "revert" or "cherry-pick" */
+		die(_("git %s: failed to read the index"), me);
 
 	prepare_revs(&revs);
 
-- 
1.7.2.3.313.gcd15

